Gems/Atom/Feature/Common/Assets/Shaders/LightCulling/LightCullingTilePrepare.azsl

```diff
@@ -66,6 +66,8 @@ void StoreTransparentMinMaxIntoSharedMemory(float2 data)
 
 ShaderResourceGroup PassSrg : SRG_PerPass
 {
+    // We will use one or the other depending upon if MSAA is enabled or not
+    Texture2D<float4> m_depthBuffer;
```

Comment:
You do not need this. Thats why we added the super variant system. It will create m_depthBuffer or  m_depthBufferMSAA for you under the hood

Reply:
Adding this to BootStrapSystemComponent should address all these issues. Basically the correct super variant is not activated for ios/android

bool isNonMsaaPipeline = (pipelineDesc.m_renderSettings.m_multisampleState.m_samples == 1);
        const char* supervariantName = isNonMsaaPipeline ? AZ::RPI::NoMsaaSupervariantName : "";
        AZ::RPI::ShaderSystemInterface::Get()->SetSupervariantName(AZ::Name(supervariantName));